ui/app/appointments/constants.js

```diff
@@ -26,7 +26,8 @@ Bahmni.Appointments.Constants = (function () {
         appointmentStatusList: [ "Scheduled", "CheckedIn", "Completed", "Cancelled", "Missed" ],
         regexForTime: /^(?:(?:1[0-2]|0?[1-9]):[0-5]\d\s*[AaPp][Mm])?$/,
         privilegeManageAppointments: 'app:appointments:manageAppointmentsTab',
-        privilegeForAdmin: 'app:appointments:adminTab'
+        privilegeForAdmin: 'app:appointments:adminTab',
```

Comment:
1. privileges like "app:appointments:adminTab" seem too UI centric to be defined. Could it be just named "app:appointments:admin" and "app:appointments:manageAppointments"? 
2. how are these privileges created? I don't see them in liquibase migration scripts for the openmrs-module-appointments omod. 
3. In the omod migrations, I  see there are scripts to create "Manage Appointments" and "Manage Appointment Services" etc. Wouldn't these privileges suffice for the UI side as well?

Reply:
1. These privileges are already existing in the production. If we change the name of these privileges, we need to write migrations to take care of the existing users/privileges. As this is not part of this issue, we can create a tech debt.
2. These privileges are created using liquibase migration scripts in bahmni core omod
3. These privileges are driven more by UI design

Reply:
1. Yes. If you are changing or introducing privileges, then the module migrations must take care of it, irrespective of whether the changes are relevant to the UI or not. I am not sure about another card for this (outside of 0.91 release), as it would then compel others to do this manually. 
2. For the privileges themselves - I definitely find them overlapping. Any specific reason to have ‘app:appointments:manageAppointments’ while there is already one ‘Manage Appointments’, similarly there exists one for admin  (irrespective of whether the admin part is from UI). Its preferable if we can have one set of privileges which could be applied to both UI and backend layers.  Else, it makes it confusing and difficult to maintain. Also the service calls must be guarded by the same set of privileges. We would look to introduce ui specific privileges if its a broader level grouping at the feature (which can be also done using roles), and if we can't leverage core module privileges (or they don't make sense).